darya_suhova/src/main/java/REVC.java

```diff
@@ -0,0 +1,18 @@
+import java.util.Scanner;
+
+public class REVC {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        String result = new StringBuilder(input)
```

Comment:
This solution requires construction of 5 additional lines, which can be suboptimal for large strings. Can it be done in different way?

Reply:
fixed. Now I'm using arrays